std/datetime/mod.ts

```diff
@@ -150,3 +149,123 @@ export function toIMF(date: Date): string {
     months[date.getUTCMonth()]
   } ${y} ${h}:${min}:${s} GMT`;
 }
+
+/**
+ * Check given year is a leap year or not.
+ * @param year year in number or Date format
+ */
+export function isLeap(year: Date | number): boolean {
```

Comment:
Is this the correct calculation of leap year? Provide reference.

Reply:
It's based on [this formula](https://docs.microsoft.com/en-us/office/troubleshoot/excel/determine-a-leap-year#formula-to-determine-whether-a-year-is-a-leap-year) from microsoft docs.

Reply:
Please provide a link in the source code

Reply:
done